chapter2/tests/t19/terraform/_modules/container/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "container_name" {
+  description = "Name of the container"
+  type        = string
+}
+
+variable "container_image" {
+  description = "Value of the name for the Docker container"
+  type        = string
+}
+
+variable "container_image_keep_locally" {
+  description = "If true, then the Docker image won't be deleted on destroy operation. If this is false, it will delete the image from the docker local storage on destroy operation."
+  type        = bool
+  default     = false
+}
+
+variable "container_ports" {
+  description = "Value of the name for the Docker container"
+  type        = map(any)
+  default = {
```

Comment:
prefer auto.tfvars and tfvars to set the values, no need to do that in the variables.tf

Reply:
adjust all vars here

Reply:
> adjust all vars here

@dmitry-mightydevops 
Done
But not sure if everything was done correctly